conda-forge.yml

```diff
@@ -1,13 +1,14 @@
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
+  osx_arm64: osx_64
+conda_build:
+  pkg_format: '2'
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
-conda_build:
-  pkg_format: '2'
-build_platform:
-  osx_arm64: osx_64
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
 provider:
   linux_aarch64: azure
   linux_ppc64le: azure
+test: native_and_emulated
```

Comment:
This part is actually useful. I didn't realize that `test: all` was the [default](https://conda-forge.org/docs/maintainer/conda_forge_yml/#test). Using this configuration setting would be cleaner than manually marking each test with `# [unix and (not (osx and arm64))]`, eg

https://github.com/conda-forge/capnproto-feedstock/blob/56102b149acf9ea36b0b9628df290bf000956f78/recipe/meta.yaml#L44-L47